src/app/beer_garden/config.py

```diff
@@ -0,0 +1,1042 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import sys
+from argparse import ArgumentParser
+from datetime import datetime
+from typing import Tuple, Sequence
+
+import six
+from ruamel.yaml import YAML
+from yapconf import YapconfSpec, dump_data
+
+from beer_garden.log import default_app_config
+
+__all__ = ["load", "generate_logging", "generate", "migrate", "get"]
+
+_CONFIG = None
+
+
+def load(args: Sequence[str], force=False) -> None:
```

Comment:
Type Annotation suggestion/question:

1. If we type annotate one thing, we should type annotate all things.
2. Should we annotate a return type of `None`?  We do here, but not for the rest of the functions.

Reply:
Oh yeah, you mean for `force`? I agree, we should do all the args.

I don't know about the `None` return type. I don't really want to go though and add type hints to everything, so I figured that we could just add them as we touch things. My thinking was that explicit was better than implicit, but I don't know how useful this particular thing is.

Reply:
I agree, I'm not sure how particularly useful it is to document a return type of `None` but, let's say, that if we decide to type a method signature that will mean the arguments as well as the return type, as a rule.